src/hotspot/share/gc/shenandoah/shenandoahFullGC.cpp

```diff
@@ -1118,10 +770,9 @@ void ShenandoahFullGC::phase2_calculate_target_addresses(ShenandoahHeapRegionSet
 
     distribute_slices(worker_slices);
 
-    size_t num_workers = heap->max_workers();
-
+    // TODO: This is ResourceMark is missing upstream.
```

Comment:
May be roll it into the refactoring PR you have open and under review at https://github.com/openjdk/jdk/pull/17894/files#diff-d58d0fe2415e3fa64f687435baf73227fb4e3e9eee8ba52ba1f3bb8301437106 ?